shared/src/business/entities/courtIssuedDocument/CourtIssuedDocumentTypeG.js

```diff
@@ -24,6 +25,18 @@ CourtIssuedDocumentTypeG.prototype.init = function init(rawProps) {
 };
 
 CourtIssuedDocumentTypeG.prototype.getDocumentTitle = function () {
+  if (this.trialLocation === TRIAL_SESSION_SCOPE_TYPES.standaloneRemote) {
```

Comment:
nit: maybe see if some of this logic can be extracted to a shared util function between these two document types.